src/main/kotlin/game/minesweeper/domain/map/Row.kt

```diff
@@ -0,0 +1,17 @@
+package game.minesweeper.domain.map
+
+data class Row(private val _fragments: List<Fragment>) {
+
+    fun fragments() = _fragments
```

Comment:
_fragment 또한, 아래 처럼 바꾸는 것이 나아 보입니다 :)
```suggestion
data class Row(val fragments: List<Fragment>) {

```